src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar
+
+import kotlin.random.Random
+
+data class Car(var defaultDistance: Int = 0) {
+    var distance = defaultDistance
+        private set
```

Comment:
```suggestion
data class Car(defaultDistance: Int = 0) {
    var distance = defaultDistance
        private set
```
내부에서는 distance를 활용하고 있으니, defaultDistance는 프로퍼티로 선언하지 않아도 돼요

Reply:
data class로 사용하고 있어서 선언이 필요한 것 같습니다